aws/nat-server.tf

```diff
@@ -1,27 +1,18 @@
 resource "aws_instance" "nat" {
-  ami = "${lookup(var.nat_ami, var.region)}"
+  ami = "${lookup(var.amis, var.region)}"
   instance_type = "t2.micro"
   subnet_id = "${aws_subnet.public.0.id}"
   security_groups = ["${aws_security_group.default.id}", "${aws_security_group.nat.id}"]
   key_name = "${var.key_pair_name}"
   source_dest_check = false
-  tags = { 
+  tags = {
     Name = "${var.env}-tsuru-nat"
   }
   connection {
-    user = "ec2-user"
+    user = "ubuntu"
     key_file = "ssh/insecure-deployer"
   }
   provisioner "remote-exec" {
-    inline = [
-      "yum update -y aws*",
-      "echo 1 | sudo tee /proc/sys/net/ipv4/ip_forward",
-      "echo 0 | sudo tee /proc/sys/net/ipv4/conf/eth0/send_redirects",
-      "sudo iptables -t nat -A POSTROUTING -o eth0 -s 0.0.0.0/0 -j MASQUERADE",
-      "sudo iptables-save | sudo tee /etc/sysconfig/iptables",
-      "sudo mkdir -p /etc/sysctl.d/",
-      "echo 'net.ipv4.ip_forward = 1' | sudo tee /etc/sysctl.d/nat.conf",
-      "echo 'net.ipv4.conf.eth0.send_redirects = 0' | sudo tee -a /etc/sysctl.d/nat.conf"
-    ]
+    script = "../setup-nat-routing.sh"
```

Comment:
I don't necessarily think that this should be reverted, but just to note, that changing this to a script prevents any output from being shown. So we don't see the additional `echo` statements that have been added and it's hard to debug whether it's doing anything when this happens:
```
aws_instance.nat: Provisioning with 'remote-exec'...
aws_instance.nat (remote-exec): Connecting to remote host via SSH...
aws_instance.nat (remote-exec):   Host: 52.17.243.1
aws_instance.nat (remote-exec):   User: ubuntu
aws_instance.nat (remote-exec):   Password: false
aws_instance.nat (remote-exec):   Private key: true
aws_instance.nat (remote-exec):   SSH Agent: false
aws_instance.nat (remote-exec): Connecting to remote host via SSH...
aws_instance.nat (remote-exec):   Host: 52.17.243.1
aws_instance.nat (remote-exec):   User: ubuntu
aws_instance.nat (remote-exec):   Password: false
aws_instance.nat (remote-exec):   Private key: true
aws_instance.nat (remote-exec):   SSH Agent: false
aws_instance.nat (remote-exec): Connecting to remote host via SSH...
aws_instance.nat (remote-exec):   Host: 52.17.243.1
aws_instance.nat (remote-exec):   User: ubuntu
aws_instance.nat (remote-exec):   Password: false
aws_instance.nat (remote-exec):   Private key: true
aws_instance.nat (remote-exec):   SSH Agent: false
aws_instance.nat (remote-exec): Connecting to remote host via SSH...
aws_instance.nat (remote-exec):   Host: 52.17.243.1
aws_instance.nat (remote-exec):   User: ubuntu
aws_instance.nat (remote-exec):   Password: false
aws_instance.nat (remote-exec):   Private key: true
aws_instance.nat (remote-exec):   SSH Agent: false
aws_instance.nat (remote-exec): Connected!
```
… several minutes pass …
```
aws_instance.nat: Error: 1 error(s) occurred:

* Script exited with non-zero exit status: 100
```
NB: If we are able to fix the output again, then we should probably quieten down the `apt-get` operations as they were before.